rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMEvent.java

```diff
@@ -136,4 +145,27 @@ public String toString() {
     public Object clone() throws CloneNotSupportedException {
         return super.clone();
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
```

Comment:
I would add brackets to the two `if` in this method.

Reply:
ok